spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/receiving-messages/class-level-kafkalistener.adoc

```diff
@@ -62,3 +62,25 @@ void listen(Object in, @Header(KafkaHeaders.RECORD_METADATA) ConsumerRecordMetad
     ...
 }
 ----
+
+Also, this won't work as well.
+The `payload` value will be injected as the `topic`.
```

Comment:
" will be injected into the `topic` parameter"?

Otherwise I ma getting confused as before.
Such a sentence makes me think that header value is injected into the `payload` parameter.